.github/workflows/sync-figma-to-tokens.yml

```diff
@@ -0,0 +1,36 @@
+name: Sync Figma to tokens
+on:
+  workflow_dispatch:
+jobs:
+  sync-figma-to-tokens:
+    runs-on: ubuntu-latest
+    permissions:
+      # Need to be able to create new branches and commits
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: '18.20.2'
+
+      - name: Clone repo
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: npm ci
```

Comment:
shouldn't this be pnpm? I don't know what kind of consequences it can have to use npm in a pnpm monorepo